src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { router } from 'expo-router';
+import React from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import SignOut from '../../../../../assets/sign-out.svg';
+import X from '../../../../../assets/x.svg';
+import { useSession } from '../../../../context/AuthContext';
+function LogOutConfirmation() {
+  const { signOut } = useSession();
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.screenContainer}>
+        <View style={styles.contentContainer}>
```

Comment:
I consolidated these containers into one. I usually try to avoid nesting divs unless they're explicitly being used for position elements on the screen.